ngraph/core/include/openvino/core/preprocess/pre_post_process.hpp

```diff
@@ -5,13 +5,13 @@
 #pragma once
 
 #include "openvino/core/core_visibility.hpp"
+#include "openvino/core/function.hpp"
+#include "openvino/core/node.hpp"
```

Comment:
Why do you need node header here?
We don't use node in this file

Reply:
@ilyachur this PR is based on another one -- https://github.com/openvinotoolkit/openvino/pull/7625 
Fixed it in the base PR, as it'll be merged, this will disappear here.